### PR TITLE
Added image save docs

### PR DESCRIPTION
**Pull request by gerth2**
Added documentation describing the image save command NT entry usage and behavior.

### CODE REVIEW

File: source/docs/programming/nt-api/nt-api.rst [hunk 1]
```diff
@@ -37,6 +37,23 @@ Changing Settings
...
+Images are returned as part of the .zip package from the "Export" operation in the Settings tab.
+
++----------------------+-------------+----------------------------------------------------+
+|         Key          |    Type     |                     Description                    |
++======================+=============+====================================================+
+| ``inputSaveImgCmd``  | ``boolean`` | Triggers saving the current input image to file.   |
++----------------------+-------------+----------------------------------------------------+
+| ``outputSaveImgCmd`` | ``boolean`` | Triggers saving the current output image to file.  |
++----------------------+-------------+----------------------------------------------------+
+
```
Comment (prateekma) [inline source/docs/programming/nt-api/nt-api.rst:56]: Will these booleans automatically be set back to false from PhotonVision or will the user have to do that?

Comment (gerth2) [reply source/docs/programming/nt-api/nt-api.rst:56]: They get set back 500ms after PhotonVision detects them to be True. User can manually toggle back faster if they want to take images more rapidly than 2/sec .

Worthwhile to add that level of detail?

Comment (prateekma) [reply source/docs/programming/nt-api/nt-api.rst:56]: Probably not, but I think it's worth mentioning that it gets set back automatically.

**Review by mcm001 — APPROVED** [review]

**Review by mcm001 — CHANGES_REQUESTED** [review]
@gerth2 wait actually, can we also detail the stuff that was added to photonlib? Specifically takeInputSnapshot and takeOutputSnapshot

**Review by mcm001 — APPROVED** [review]

**Pull request merged**